aten/src/ATen/native/mkldnn/xpu/FusionUtils.cpp

```diff
@@ -0,0 +1,79 @@
+#include <ATen/native/mkldnn/xpu/FusionUtils.h>
+
+using namespace at::native::onednn;
+
+namespace at::native::xpu {
+
+onednn::Attr unary_attr_with_arg(
+    std::string_view unary,
+    torch::List<std::optional<at::Scalar>> scalars =
+        torch::List<std::optional<at::Scalar>>(),
+    std::optional<std::string_view> algorithm = std::nullopt,
```

Comment:
In general, default values are typically specified in the function declaration, most often in the header file.

Reply:
the argument list is too much complex with default value. I did a little order change(move attr to 1st) and lots of overloading syntax error. So, I decides the remove all default arguments in these utils.